Mathlib/LinearAlgebra/DFinsupp.lean

```diff
@@ -648,6 +649,18 @@ theorem map_dfinsupp_sumAddHom [∀ i, AddZeroClass (γ i)] (f : M ≃ₛₗ[τ
     f (sumAddHom g t) = sumAddHom (fun i => f.toAddEquiv.toAddMonoidHom.comp (g i)) t :=
   f.toAddEquiv.map_dfinsupp_sumAddHom _ _
 
+/-- Given a linear equivalence `f : M ≃ₛₗ[τ₁₂] M₂[σ] N` and a function `g : α → M`, we have
+`f ∑ᶠ g i = ∑ᶠ f (g i)`.  -/
+theorem map_finsum (f : M ≃ₛₗ[τ₁₂] M₂) (g : α → M) :
+    f (finsum fun i : α ↦ g i) = finsum fun i : α ↦ f (g i) :=
+  AddEquiv.map_finsum f.toAddEquiv g
```

Comment:
Looks like this should be written for `AddEquivClass` instead

Reply:
Do you have a suggestion of how to do that? Looking at Mathlib, it seems like these kind of lemmas are currently available for Equiv/Homs, but not for EquivClass/HomClass (e.g., AddEquiv.map_finsum, MonoidHom.map_finprod_of_injective, MonoidHom.map_finprod_of_preimage_one).

Reply:
For now I have just generalized these two lemmas to `MulEquivClass` (and additivized), but this still does not seem very consistent with the rest of the API.